spec/queries/entitlement/subscription_entitlement_query_spec.rb

```diff
@@ -0,0 +1,378 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Entitlement::SubscriptionEntitlementQuery, type: :service, transaction: false do
```

Comment:
Any reason to prevent the transactional specs here?

Reply:
Thank you! I was debugging and I wanted the seeder to write the data so I could check it in TablePlus. I forgot to remove 🙏